src/components/GradientList.vue

```diff
@@ -45,18 +44,53 @@
             :key="'copied'">Copied!</div>
         </transition>
       </button>
+      <form
+        class="export-form"
+        action="https://codepen.io/pen/define"
+        method="POST"
+        target="_blank">
+        <input
+          type="hidden"
+          name="data"
+          :value="getFormData">
+        <button
+          type="submit"
+          class="btn btn-export btn-shadow">
+          Export to CodePen
+        </button>
+      </form>
+
     </li>
   </ul>
 </template>
 
 <script>
 import { mapGetters } from 'vuex'
+import { createHelpers } from 'vuex-map-fields'
+const { mapFields } = createHelpers({
+  getterType: 'canvas/getField'
+})
+
 export default {
   data: () => ({
     copied: false
   }),
   computed: {
-    ...mapGetters(['gradientList', 'previewGradient', 'gradientStrings'])
+    ...mapFields(['canvas']),
+    ...mapGetters(['gradientList', 'previewGradient', 'gradientStrings']),
+    getFormData() {
+      return JSON.stringify({
+        title: 'Shapy Gradient 🤖',
+        html: '<div class="gradient"></div>',
+        css: `
+body, html {width: 100%; height: 100%}
```

Comment:
This needs to be here because the default is 100% by 100% and if I don't set this the gradient won't even show :(